Orange/widgets/data/owdiscretize.py

```diff
@@ -245,6 +245,9 @@ def _intbox(parent, attr, callback):
 
         self._update_spin_positions()
 
+        self.info.set_output_summary(self.info.NoInput)
```

Comment:
When setting output summary, output default message (`self.info.NoOutput`) should be used.